app/views/draftmetadata/draftMetadataChecksErrorsNoDownload.scala.html

```diff
@@ -0,0 +1,20 @@
+@import services.FileError
+@import views.html.partials._
+
+@import java.util.UUID
+@(consignmentId: UUID, consignmentRef: String, draftMetadataProgress: DraftMetadataProgress, name: String, actionMessage:String, detailsMessage:String)(implicit request: RequestHeader, messages: Messages)
+
+@main("Results of CSV Checks", name = name) {
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            @draftMetadataChecksActionProcess(actionMessage, detailsMessage)
+            <p class="govuk-body">Once you have addressed this issue upload a revised metadata file.</p>
+
+            <div class="govuk-button-group">
+                <a class="govuk-button" href="@{routes.AdditionalMetadataEntryMethodController.additionalMetadataEntryMethodPage(consignmentId)}" role="button" draggable="false" data-module="govuk-button">Re-upload metadata</a>
```

Comment:
It should take to the metadata upload page instead of the entry page: `routes.DraftMetadataUploadController.draftMetadataUploadPage(consignmentId)`